src/NServiceBus.PersistenceTesting/IPersistenceTestsConfiguration.cs

```diff
@@ -37,25 +37,27 @@ public interface IPersistenceTestsConfiguration
 
         SagaMetadataCollection SagaMetadataCollection { get; }
 
-        Task Configure();
+        Task Configure(object testClass);
```

Comment:
is this necessary given the variant is already available as a property?https://github.com/Particular/NServiceBus/pull/5687/files#diff-06a081f26d8e4f1183fdd1f4ac8e0ab2R60

Reply:
Yeah. The value passed here is not the variant but the actual test fixture object. This is because of a limitation in SQLP where the name of the saga class can be generated based on the name of the saga which does not include the declaring type name in case of nested classes. But actually I think I found a way to slightly modify SQLP to get around this.